content/ja/about.md

```diff
@@ -12,26 +12,28 @@ NumPy は 、NumPyコミュニティやより広範な科学計算用Python コ
 
 ## 運営委員会
 
-NumPy運営委員会の役割は、NumPyのコミュニティと協力しサポートすることを通じて、技術的にもコミュニティ的にも長期的にNumPyプロジェクトを良い状態に保つことです。 NumPy運営委員会は現在以下のメンバーで構成されています (アルファベット順):
+NumPy運営委員会の役割は、NumPyのコミュニティと協力しサポートすることを通じて、技術的にもコミュニティ的にも長期的にNumPyプロジェクトを良い状態に保つことです。 NumPy運営委員会は現在以下のメンバーで構成されています (アルファベット順、姓で):
 
 - Sebastian Berg
-- Jaime Fernández del Río
 - Ralf Gommers
-- Allan Haldane
 - Charles Harris
 - Stephan Hoyer
+- Melissa Weber Mendonça
```

Comment:
Well, there's the issue... Weber is a last name (my mother's side) so technically I should go after Stéfan I think? 

Otherwise, looks good :)

Reply:
Excellent point, moved things around ^_^